macros/src/lib.rs

```diff
@@ -1131,7 +1131,7 @@ impl Codegen {
             .iter()
             .filter_map(|frag| match frag {
                 Fragment::Parameter(param) => Some(param.clone()),
-                Fragment::Literal(_) => None,
+                _ => None,
```

Comment:
This was exhaustive to cause a compiler error when new variants are added

Reply:
This makes sense and I didn't think about that. Iirc there was a second place where I did this.

Reply:
Seems to be only this one. Should be good now!